docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -19,8 +19,8 @@ order of Pods and out-of-resource eviction ordering on the Node.
 
 {% capture body %}
 
-## How to use priority and preemption
-To use priority and preemption in Kubernetes 1.8 and later, follow these steps:
+## How to use priority
```

Comment:
We are still 100% compatible with the previous versions and those instruction will still work as is in 1.11. So, I would leave all these parts as they are and just add a new section on "How to disable preemption" and explain how what flag should be set to disable preemption in 1.11.

Reply:
Thanks, just fixed